package-lock.json

```diff
@@ -2941,6 +2941,11 @@
 				"uuid": "^3.2.1"
```

Comment:
We don't need to update this file, please undo it.

Reply:
it is not updated as i can see on Rocket chat github it is same as mine